src/settings/LoanHistory/ExceptionsList.js

```diff
@@ -0,0 +1,81 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import {
+  uniqBy,
+  get,
+} from 'lodash';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { Button } from '@folio/stripes/components';
+
+import ExceptionCard from './ExceptionCard';
+
+import css from './ExceptionsList.css';
+
+@stripesConnect
+class ExceptionsList extends Component {
+  static manifest = Object.freeze({
+    paymentMethods: {
+      type: 'okapi',
+      path: 'payments',
+      throwsErrors: false,
+    },
+  });
+
+  static propTypes = {
+    fields: PropTypes.object.isRequired,
+    resources: PropTypes.shape({
+      paymentMethods: PropTypes.object,
+    }).isRequired,
+  }
+
+  onAddField = () => {
```

Comment:
Let's name the methods which we pass to other components like without `on` part but with rather `handle` or other **verb**, so `handleAddField`, so the receiver is named. See [examples](https://reactjs.org/docs/handling-events.html). Check other places (e.g. `onRemoveField`).

Eventually the way below is preferable
`onClick={this.handleAddField}`.